packages/api-explorer/__tests__/Params.test.jsx

```diff
@@ -58,8 +58,8 @@ test('boolean should render as <select>', () => {
   const select = params.find('.field-boolean select');
 
   expect(select.length).toBe(1);
-  expect(select.find('option').length).toBe(3);
-  expect(select.find('option').map(el => el.text())).toEqual(['', 'true', 'false']);
+  expect(select.find('option').length).toBe(2);
```

Comment:
This has been fixed in master now! It was due to an upstream change in the form module. Re-merging master will fix this test. Here's the commit: https://github.com/readmeio/api-explorer/commit/9d2e001346774a6792861bebad3b2618f123d6fc

Reply:
I've reverted this test and merged master into this branch.